compiler/ras/Debug.hpp

```diff
@@ -141,7 +141,6 @@ namespace TR { class X86FPMemRegInstruction;               }
 namespace TR { class X86FPRegMemInstruction;               }
 class TR_X86RegisterDependencyGroup;
 namespace TR { class X86RestartSnippet; }
-namespace TR { class S390LookupSwitchSnippet; }
```

Comment:
Could you please elaborate on what a target address snippet or lookup switch snippet was envisaged for ? I know these concepts are not being used in the codebase currently (hence your deletion of this code) but a different question is whether they could be useful down the line. Is there a short reason why you feel the notions are themselves not useful ?

Reply:
It seems to have targeted a key/value switch table, likely to compare against certain keys and jump to certain addresses. It required some infrastructure from a specific node which as far as I can tell no longer exists in OMR.